From 06862133951b8efc9ef0d90788fa54db24f55ec8 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Wed, 24 May 2017 22:45:36 +0800
Subject: [PATCH 1810/5242] MLK-14947-3 usb: doc: chipidea: imx: add
 power-polarity-active-high property

commit  a1966dd02392bdc4f9b5f1836c2701bd77caa3ea from
https://source.codeaurora.org/external/imx/linux-imx.git

USB host vbus control can be via port power(PP) bit of ehci, there
is a polarity setting in controller register for this signal, if
power supply chip use active high, add this property.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/usb/ci-hdrc-usb2.txt       |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
index 7ffc3e9..3b7ba44 100644
--- a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
+++ b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
@@ -100,6 +100,9 @@ i.mx specific properties
 - osc-clkgate-delay: the delay between powering up the xtal 24MHz clock
   and release the clock to the digital logic inside the analog block,
   0 <= osc-clkgate-delay <= 7.
+- power-polarity-active-high: add this property if port power function of ehci
+  is used to enable vbus, and the vbus power supply chip enable signal is high
+  active.
 
 Examples:
 
-- 
1.7.9.5

